### PR TITLE
Show pt type in pt listing mtm #152

### PR DESCRIPTION
**Pull request by schangccg**

### CODE REVIEW

**Comment by schangccg** [comment]
For ticket: https://github.com/muccg/mtm/issues/152

File: rdrf/rdrf/patients_listing.py [hunk 1]
```diff
@@ -84,8 +84,7 @@ def get_columns(self):
         return [
             ColumnFullName(_("Patient"), "patients.can_see_full_name"),
             ColumnDateOfBirth(_("Date of Birth"), "patients.can_see_dob"),
-            ColumnPatientType(_("Type"), "patients.can_see_type"),
-            ColumnPatientGender(_("Gender"), "patients.can_see_gender"),
+            ColumnCodeField(_("Code Field"), "patients.can_see_code_field"),
```
Comment (id2359) [inline rdrf/rdrf/patients_listing.py]: This is actually a custom permission name - We probably haven't created a custom permission object for this yet 
I'll fix up in my branch though

File: rdrf/registry/patients/models.py [hunk 2]
```diff
@@ -261,6 +261,28 @@ class Meta:
...
+    def code_field(self):
+        gender_string = ""
+
+        if self.sex == "1":
+            gender_string = "Male"
+        elif self.sex == "2":
+            gender_string = "Female"
+        elif self.sex == "3":
+            gender_string = "Indeterminate"
+        else:
```
Comment (id2359) [inline rdrf/registry/patients/models.py]: you could use a dictionary here

gender_map = {"1": "Male",...}

Also we need translation marker  _("Male")

**Comment by id2359** [comment]
Approved will merge into mtm-phase2 prior to next_release

**Pull request closed without merging**